src/style.css

```diff
@@ -0,0 +1,77 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
```

Comment:
can you achieve the same without `flex-grow`?

Reply:
you really only need `flex-basis`
less is more

Reply:
Entire_document_750. & Entire_document_450  fail without flex-grow